### ISSUE TITLE
function_parameter_count should be able to warn(/show error) for multiline functions

### ISSUE THREAD

**Issue by Sajjon** [realm/SwiftLint#674]
It would be nice if it would be possible to configure the function_parameter_count rule to trigger errors/warnings even for functions declaring their parameters on separate lines. Or is it already possible?

``` swift
    // using "error: 4" for rule "function_parameter_count" triggers error for this method
    func triggersErrors(a: Int, b: Int, c: Int, d: Int, e: Int) {
        print("this method triggers error")
    }

    // using "error: 4" for rule "function_parameter_count" allows this
    func doesntTriggerError(a: Int,
                      b: Int,
                      c: Int,
                      d: Int,
                      e: Int)
    {
        print("this method does not trigger any error")
    }
```

Thanks!

**Comment by marcelofabri** [comment]
I've tested this and couldn't reproduce. Is it still an issue, @Sajjon?

```
$ echo "func triggersErrors(a: Int, b: Int, c: Int,\nd: Int, e: Int, f: Int) { }" | swiftlint lint --use-stdin 

Loading configuration from '.swiftlint.yml'
<nopath>:1:1: warning: Function Parameter Count Violation: Function should have 5 parameters or less: it currently has 6 (function_parameter_count)
```

**Comment by jpsim** [comment]
I think this has been fixed, since I also can't reproduce.
